web-components/src/mixins/FocusTrapMixin.ts

```diff
@@ -177,13 +177,23 @@ export const FocusTrapMixin = <T extends AnyConstructor<FocusClass & FocusTrapCl
       return false;
     }
 
+    private shouldSkipFocus(element: HTMLElement) {
+      if(element.className.split(" ").indexOf('md-combobox')>-1){
```

Comment:
please write a comment, why we need to skip child element of dropdown

Reply:
Added